test/e2e/pkg/testnet.go

```diff
@@ -74,15 +74,14 @@ const (
 // Testnet represents a single testnet.
 // It includes all fields from the associated Manifest instance.
 type Testnet struct {
-	Manifest
+	*Manifest
```

Comment:
Are you sure this doesn't break anything?

Reply:
It was breaking when not used as a pointer, because we would copy some fields (non-pointer fields) but not others. A total mess.
As we are now modifying the Manifest down below in this function, those modifications didn't propagate (hence I needed to turn it into a pointer)